Shared/mods/deathmatch/logic/luadefs/CLuaCryptDefs.cpp

```diff
@@ -368,12 +368,19 @@ int CLuaCryptDefs::EncodeString(lua_State* luaVM)
     StringEncryptFunction algorithm;
     SString               data;
     CStringMap            options;
+    CLuaFunctionRef       luaFunctionRef;
 
     CScriptArgReader argStream(luaVM);
     argStream.ReadEnumString(algorithm);
     argStream.ReadString(data);
     argStream.ReadStringMap(options);
 
+    if (argStream.NextIsFunction())
+    {
+        argStream.ReadFunction(luaFunctionRef);
```

Comment:
You can do `argStream.ReadFunction(iLuaFunction, LUA_REFNIL);` and then later on, instead of `if (luaFunctionRef == CLuaFunctionRef())` you can do `if (VERIFY_FUNCTION(luaFunctionRef))`

Reply:
Thank you. I just copied the code from the passwordHash function 😁

Reply:
Yep 😆, when looking up existing uses of ReadFunction I noticed that `passwordHash` was the only one that does that!